config/config.go

```diff
@@ -210,6 +210,7 @@ func ReadNetclientConfig() (*Config, error) {
 		return nil, err
 	}
 	defer f.Close()
+	netclient = Config{}
```

Comment:
I believe decoding will override the variable, I don't think this is the root cause of the issue

Reply:
It is a valid assumption, same as mine, but it turns out that yaml...Decode adds config entries to an existing struct.

When the previous config has `peers`, new config is without `peers` is read, the old `peers` remain untouched. Re-initing the struct fixed that.

Reply:
interesting,  I had the same assumption